examples/xload/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/gojekfarm/xtools/xload"
+)
+
+func businessConfigLoader() xload.Loader {
+	return xload.LoaderFunc(func(ctx context.Context, key string) (string, error) {
+		// fetch from remote config
+		return "", nil
+	})
+}
+
+func yamlLoader(filename string) xload.Loader {
+	// read into a flattened map
+	// return a MapLoader
+	return xload.MapLoader{}
+}
+
+func main() {
+	ctx := context.Background()
+
+	// application config is loaded once at startup
+	// from yaml, and then from environment variables
+	cfg := newAppConfig()
+
+	err := xload.Load(
+		ctx,
+		cfg,
+		xload.WithLoader(
```

Comment:
Does it require Option `xload.Key("env")`

Reply:
yup. Added it now